txpipe/rail_pz.py

```diff
@@ -0,0 +1,190 @@
+from .base_stage import PipelineStage
+from .data_types import PhotozPDFFile, HDFFile, PickleFile
+import numpy as np
+
+
+class PZRailTrain(PipelineStage):
+    """Train a photo-z model using RAIL.
+
+    The Redshift Assessment Infrastructure Layers (RAIL) library provides a uniform
+    interface to DESC photo-z code.
+
+    TXPipe uses RAIL across several different pipeline stages.
+    This stage, which would normally be run first, uses a training set (e.g. of
+    spectroscopic redshifts) to train and save an Estimator object that a later
+    stage can use to measure redshifts of the survey sample.
+    """
+    name = "PZRailTrain"
+
+    inputs = [
+        ("photoz_training", HDFFile),
+        ("photoz_testing", HDFFile),
+    ]
```

Comment:
is there a way to be more agnostic at this stage, about what format the inputs will actually be in?

Reply:
That's a good idea - I've changed them to use the generic `DataFile` type, which can be anything.